esp-link/cgiwifi.c

```diff
@@ -48,100 +56,100 @@ static char *wifiPhy[]  = { 0, "11b", "11g", "11n" };
 void (*wifiStatusCb)(uint8_t); // callback when wifi status changes
 
 static char* ICACHE_FLASH_ATTR wifiGetReason(void) {
-  if (wifiReason <= 24) return wifiReasons[wifiReason];
-  if (wifiReason >= 200 && wifiReason <= 201) return wifiReasons[wifiReason-200+24];
-  return wifiReasons[1];
+    if (wifiReason <= 24) return wifiReasons[wifiReason];
+    if (wifiReason >= 200 && wifiReason <= 201) return wifiReasons[wifiReason-200+24];
+    return wifiReasons[1];
 }
 
 // handler for wifi status change callback coming in from espressif library
 static void ICACHE_FLASH_ATTR wifiHandleEventCb(System_Event_t *evt) {
-  switch (evt->event) {
-  case EVENT_STAMODE_CONNECTED:
-    wifiState = wifiIsConnected;
-    wifiReason = 0;
-    DBG("Wifi connected to ssid %s, ch %d\n", evt->event_info.connected.ssid,
-      evt->event_info.connected.channel);
-    statusWifiUpdate(wifiState);
-    break;
-  case EVENT_STAMODE_DISCONNECTED:
-    wifiState = wifiIsDisconnected;
-    wifiReason = evt->event_info.disconnected.reason;
-    DBG("Wifi disconnected from ssid %s, reason %s (%d)\n",
-      evt->event_info.disconnected.ssid, wifiGetReason(), evt->event_info.disconnected.reason);
-    statusWifiUpdate(wifiState);
-    break;
-  case EVENT_STAMODE_AUTHMODE_CHANGE:
-    DBG("Wifi auth mode: %d -> %d\n",
-      evt->event_info.auth_change.old_mode, evt->event_info.auth_change.new_mode);
-    break;
-  case EVENT_STAMODE_GOT_IP:
-    wifiState = wifiGotIP;
-    wifiReason = 0;
-    DBG("Wifi got ip:" IPSTR ",mask:" IPSTR ",gw:" IPSTR "\n",
-      IP2STR(&evt->event_info.got_ip.ip), IP2STR(&evt->event_info.got_ip.mask),
-      IP2STR(&evt->event_info.got_ip.gw));
-    statusWifiUpdate(wifiState);
-    if (!mdns_started)
-      wifiStartMDNS(evt->event_info.got_ip.ip);
-    break;
-  case EVENT_SOFTAPMODE_STACONNECTED:
-    DBG("Wifi AP: station " MACSTR " joined, AID = %d\n",
-        MAC2STR(evt->event_info.sta_connected.mac), evt->event_info.sta_connected.aid);
-    break;
-  case EVENT_SOFTAPMODE_STADISCONNECTED:
-    DBG("Wifi AP: station " MACSTR " left, AID = %d\n",
-        MAC2STR(evt->event_info.sta_disconnected.mac), evt->event_info.sta_disconnected.aid);
-    break;
-  default:
-    break;
-  }
-
-  for (int i = 0; i < 4; i++) {
-    if (wifi_state_change_cb[i] != NULL) (wifi_state_change_cb[i])(wifiState);
-  }
+    switch (evt->event) {
```

Comment:
I know, I am new to git-hub. The main problem was that the files are unreadable in Xcode, so I had to change the indentation to understand the code. I'll try visual studio..